components/kyma-environment-broker/internal/process/deprovisioning/remove_runtime.go

```diff
@@ -74,7 +74,7 @@ func (s *RemoveRuntimeStep) Run(operation internal.DeprovisioningOperation, log
 		}
 	}
 
-	log.Infof("runtime deletion process initiated successfully, RuntimeID=%s", instance.RuntimeID)
+	log.WithField("runtimeID", instance.RuntimeID).Infof("runtime deletion process initiated successfully")
```

Comment:
create a logger with this field when we know there is a runtimeid, then use it. Look, in line 64, 68 you there are logging statement, but runtimes won't be there